### PR TITLE
[TACHYON-1303] Add unit tests for TachyonFile

### PR DESCRIPTION
**Pull request by shuhuai007**
Please review:
https://tachyon.atlassian.net/browse/TACHYON-1303

### CODE REVIEW

**Comment by gpang** [comment]
LGTM

**Comment by yupeng9** [comment]
Thanks for the test improvement!

File: clients/unshaded/src/test/java/tachyon/client/file/TachyonFileTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  @Test
+  public void getFileIdTest() {
+    Assert.assertEquals(100, new TachyonFile(100).getFileId());
+  }
+
+  @Test
+  public void toStringTest() {
+    Assert.assertEquals("TachyonFile(100)", new TachyonFile(100).toString());
+  }
+
```
Comment (calvinjia) [inline clients/unshaded/src/test/java/tachyon/client/file/TachyonFileTest.java]: Extra space?

**Comment by yupeng9** [comment]
LGTM

File: clients/unshaded/src/test/java/tachyon/client/file/TachyonFileTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Tests {@link TachyonFile}.
+ */
+public final class TachyonFileTest {
+
+  @Test
+  public void equalsTest() {
```
Comment (calvinjia) [inline clients/unshaded/src/test/java/tachyon/client/file/TachyonFileTest.java]: Could you add a short comment about what each test covers?

**Comment by calvinjia** [comment]
@shuhuai007 Thanks for the patch, I will address my formatting comments in a follow up commit.

**Pull request merged**